pyvista/examples/planets.py

```diff
@@ -0,0 +1,739 @@
+"""Downloadable datasets of 3D Celestial Bodies.
+
+Examples
+--------
+>>> from pyvista import examples
+>>> mesh = examples.planets.load_moon()
+>>> mesh.plot()
+
+"""
+
+import numpy as np
+
+import pyvista
+from pyvista import examples
+
+from .downloads import _download_and_read
+
+
+def _sphere_with_texture_map(radius=1.0, theta_resolution=50, phi_resolution=100):
+    """Sphere with texture coordinate.
+
+    Parameters
+    ----------
+    radius : float, optional
+        Sphere radius.
+
+    theta_resolution : int , optional
+        Set the number of points in the longitude direction.
```

Comment:
The way this is defined now, theta is the polar angle and phi is the azimuthal angle. In other words, theta is the latitudal direction and phi the longitudal.

Reply:
```suggestion
        Set the number of points in the latitudinal direction.
```

Reply:
Well, it might not be a bad idea to keep that with the API... I just always default to the physicist's definition of polar coordinates :) So perhaps we could flip the meaning of `theta_resolution` and `phi_resolution`, probably with flipping theta <-> phi everywhere in the function. Then everything is consistent. (I know this means re-rewriting a lot of stuff; sorry.)

Reply:
Okay, so I would like to make Sphere As is. How about we follow the physics convention for this function and add a Warning to the docstring if the argument is different from Sphere?

Reply:
That works too I think :) Middle ground: keep the current implementation, but rename `theta_resolution` to `lat_resolution` and `phi_resolution` to `lon_resolution`.

Reply:
Resolved in 91f4afa .